packages/hash/frontend/src/pages/[account-slug]/entities/[entity-id].page/entity-editor/links-section/link-table/use-create-get-cell-content.ts

```diff
@@ -0,0 +1,36 @@
+import { GridCell, GridCellKind, Item } from "@glideapps/glide-data-grid";
+import { useCallback } from "react";
+import { RowData } from "../../../../../../../components/GlideGlid/glide-grid";
+import { linkGridIndexes } from "./constants";
+import { LinkRow } from "./types";
+
+export const useCreateGetCellContent = () => {
+  const createGetCellContent = useCallback(
+    (rowData: RowData) =>
+      ([col, row]: Item): GridCell => {
+        const link = rowData[row] as LinkRow;
```

Comment:
Is it possible to use generics at all to make it so this doesn't need to be cast?